1.1.0/jessie/build-projectjson/README.md

```diff
@@ -52,6 +76,6 @@ This image contains:
     $ docker cp build-cont:/out ./output
     ```
 
-Now you have built your ASP.NET Core application inside a container and have the published output on the host ready to
-deploy. From here you could then construct an optimized runtime image with the `microsoft/aspnetcore` image or just
-deploy/run the binaries as normal without using Docker at runtime.
\ No newline at end of file
+Now you have built your ASP.NET Core application inside a container and have the published output on the host ready to deploy. From here you could then construct an optimized runtime image with the `microsoft/aspnetcore` image or just deploy/run the binaries as normal without using Docker at runtime.
```

Comment:
This does that as well, in the output directory is a fully published app that you can run with `dotnet run`. I will try to make that clearer.